starting on hippo001 (#1)
hemilabs/hippos: README.md

# Hemi Improvement ProPOsals (HIPPOS)

Largely modeled after Bitcoin's
[BIPs](https://github.com/bitcoin/bips), this repo documents changes
to the Hemi Network.

Initially, this will be light on process and mostly descriptive.  We
hope to eventually expand it into a true collaborative space to drive
Hemi forward.

| Number | Title | Type | Status |
|--------|-------|------|--------|

> | [1](./hippo-001.md) | HIPPO Purpose and Process | Process | Active |

Reviewer: The status in HIPPO-001 is currently `New`

```suggestion
| [1](./hippo-001.md) | HIPPO Purpose and Process | Process | New |
```

Author: Going to mark this as `Active` in both place then since the plan is to start following this as soon as it goes in.  As the HIPPO that defines HIPPOs it is kind of inherently circular that way.

Largely moddeled after Bitcoin's [BIPs](https://github.com/bitcoin/bips), this repo documents changes to the Hemi Network.

> Initially this will be very light on process and mostly descriptive although we hope to eventually expand this to a true collaborative space to drive Hemi forward.

Reviewer: ```suggestion
Initially, this will be light on process and mostly descriptive, though we hope to eventually expand it into a truly collaborative space to drive Hemi forward.
```

Also, in other repositories we generally wrap markdown at ~120 characters, is it worth doing that here too?

Author: Agreed on wrapping at something fairly short.

| Number | Title | Type | Status |
|--------|-------|------|--------|

> | [1](./hoppo001.md) | HIPPO Purpose and Process | Process | Active |

Reviewer 2: ```suggestion
Initially this will be very light on process and mostly descriptive although we hope to eventually expand this to a true collaborative space to drive Hemi forward.

| Number | Title | Type | Status |
|--------|-------|------|--------|
| [1](./hoppo001.md) | HIPPO Purpose and Process | Process | Active |
```

> | [1](./hoppo001.md) | HIPPO Purpose and Process | Process | Active |

Reviewer 2: ```suggestion
| [1](./hippo001.md) | HIPPO Purpose and Process | Process | Active |
```

🦛
